README.md

```diff
@@ -238,7 +238,13 @@ Neither `a.b` nor `a?.b` is intended to preserve arbitrary information on the ba
 
 In particular, the value `null` is considered to have no properties; therefore, `(null)?.b` is undefined.
 
+<dt>Why does foo?.() throw when foo is neither nullish nor callable?
```

Comment:
Don't these need closing tags?

Reply:
Also, I'd use backquotes for code examples even in headers, but I can see that's not done elsewhere in the FAQ

Reply:
i believe html auto-adds them (li’s too)

Reply:
Yeah, I mimicked the surroundings in not using closing tags, though it made me itch to do so. :joy: Happy to add them throughout if desired.

Reply:
> i believe html auto-adds them (li’s too)

🤯

Reply:
Alright, I've cleaned things up in this area accordingly. :grin: